.lintr.R

```diff
@@ -0,0 +1,12 @@
+library(lintr)
+
+source("inst/lintr/linters.R")
+
+linters <- admiral_linters()
+
+exclusions <- list(
+  "R/data.R" = Inf,
+  "inst" = list(undesirable_function_linter = Inf),
+  "R/assertions.R" = list(undesirable_function_linter = Inf),
```

Comment:
Could be removed if we use `symbol_is_undesirable = TRUE`.

Reply:
Sorry, I meant `symbol_is_undesirable = FALSE`.

Reply:
ah yes thanks, fixed.